osu.Game.Rulesets.Mania/Skinning/Argon/ManiaArgonSkinTransformer.cs

```diff
@@ -139,11 +139,11 @@ private Color4 getColourForLayout(int columnIndex, StageDefinition stage)
                 case 3:
                     switch (columnIndex)
                     {
-                        case 0: return colour_pink;
+                        case 0: return colour_green;
 
-                        case 1: return colour_orange;
+                        case 1: return colour_purple;
```

Comment:
I believe that if the goal is to match the linked image, then this should be

```suggestion
                        case 1: return colour_special_column;
```

Reply:
Kind of a weird case because it's not a "special" key. But given it's in the centre let's go with it.